packages/rollups-node/Dockerfile

```
@@ -68,7 +68,9 @@ COPY --from=inspect_server  /opt/cartesi/bin/inspect-server       /opt/cartesi/b
 COPY --from=graphql_server  /opt/cartesi/bin/graphql-server       /opt/cartesi/bin/graphql-server
 COPY --from=deployments     /opt/cartesi/share/deployments/       /opt/cartesi/share/deployments/
 
-RUN mkdir -p /var/opt/cartesi/machine-snapshots/0_0 && ln -s /var/opt/cartesi/machine-snapshots/0_0 /var/opt/cartesi/machine-snapshots/latest
+RUN mkdir -p /var/opt/cartesi/machine-snapshots/0_0 && \
+    ln -s /var/opt/cartesi/machine-snapshots/0_0 /var/opt/cartesi/machine-snapshots/latest && \
+    chown -R cartesi:cartesi /var/opt/cartesi/machine-snapshots
```

Since we're breaking lines here, I suggest using the `<<EOF` style, to keep consistency in the file.

```suggestion
RUN <<EOF
mkdir -p /var/opt/cartesi/machine-snapshots/0_0
ln -s /var/opt/cartesi/machine-snapshots/0_0 /var/opt/cartesi/machine-snapshots/latest
chown -R cartesi:cartesi /var/opt/cartesi/machine-snapshots
EOF
```